azkaban-webserver/src/main/java/azkaban/webapp/servlet/ScheduleServlet.java

```diff
@@ -648,10 +649,25 @@ private void ajaxScheduleFlow(HttpServletRequest req,
     }
 
     ReadablePeriod thePeriod = null;
+    String cronExpression = null;
+
     try {
       if (hasParam(req, "is_recurring")
           && getParam(req, "is_recurring").equals("on")) {
-        thePeriod = Schedule.parsePeriodString(getParam(req, "period"));
+        if (hasParam(req, "period")) {
+          thePeriod = Schedule.parsePeriodString(getParam(req, "period"));
+        }
+        if (hasParam(req, "cronExpression")) {
+          // everything in Azkaban functions is at the minute granularity, so we add 0 here
```

Comment:
" everything in Azkaban functions is at the minute granularity"
How about:
The scheduling thread currently checks  every minute. Thus the current implementation doesn't support sub-minute granularity in schedules.